assets/tests/components/notifications.test.tsx

```diff
@@ -26,17 +25,20 @@ jest.mock("../../src/laboratoryFeatures", () => ({
   default: () => true,
 }))
 
+const now = new Date(123_456_789)
```

Comment:
Hmm, maybe call this something different just to avoid someone looking at the file from confusing it with the `now()` helper function?